options/cf_options.h

```diff
@@ -313,6 +315,7 @@ struct MutableCFOptions {
 
   uint64_t sample_for_compression;
   std::vector<CompressionType> compression_per_level;
+  uint32_t max_tombstones_count_;
```

Comment:
I tried patching this PR and run rocksdb. In the logs, I didn't see this option being printed out with other options. Is there anything missing?

Reply:
I didn't updated the options/ , as we use Java api.
I'll update options.